src/Commands/stubs/Auth/ForgotPasswordController.stub

```diff
@@ -3,34 +3,34 @@
 namespace {{namespace}};
 
 use App\Mail\Restify\Auth\ForgotPasswordMail;
+use App\Models\User;
 use Illuminate\Http\Request;
 use Illuminate\Routing\Controller;
 use Illuminate\Support\Facades\Mail;
 use Illuminate\Support\Facades\Password;
-use App\Models\User;
 
 class ForgotPasswordController extends Controller
 {
     public function __invoke(Request $request)
     {
         $request->validate([
             'email' => ['required', 'email'],
+            'url' => ['sometimes', 'string'],
```

Comment:
https://laravel.com/docs/8.x/validation#rule-url
```suggestion
            'url' => ['sometimes', 'url'],
```
Or you can even go with `active_url` if you know it's pingable from outside, but might be a bit too much

Reply:
Test would fail, as I'm sending url with placeholders.